packages/framework/build.xml

```diff
@@ -309,6 +311,8 @@
         </exec>
     </target>
 
+    <target name="demo-data" depends="production-protection,db-wipe-public-schema,db-import-basic-structure,db-migrations,domains-data-create,db-fixtures-demo,plugin-demo-data-load,friendly-urls-generate,domains-urls-replace,elasticsearch-export" description="Wipes Postgres DB, recreates structure, fills with demo data and then exports data to Elasticsearch"/>
```

Comment:
Now there is no separate target that will only allow running demo data. I see this as limiting for common scenarios.

The other thing is that with Postgres, structure creation and migrations are included, but not for Elasticsearch, which is not consistent.

Reply:
I find it correct, that there is no target for running demo data creation only in Postgres, as it leads into faulty application, as you show different data on frontend than the data present in database.

`phing elasticsearch-export` does also migrations if needed see `IndexFacade::export` for more information.

Reply:
I would also appreciate the separate (even hidden) target for load data to database only. 
I use it on purpose when I know I don't need elastic. 

But the final decision is yours of course.

Reply:
I have hide db-demo, but kept it as it was